src/app/space/app-launcher/services/app-launcher-project-progress.service.ts

```diff
@@ -21,22 +19,13 @@ export class AppLauncherProjectProgressService implements ProjectProgressService
       if (this.END_POINT.indexOf('https') !== -1) {
         this.END_POINT = this.END_POINT.replace('https', 'wss');
       } else if (this.END_POINT.indexOf('http') !== -1) {
-        this.END_POINT = this.END_POINT.replace('http', 'wss');
+        this.END_POINT = this.END_POINT.replace('http', 'ws');
       }
     }
   }
 
   getProgress(uuidLink: string): WebSocket {
-    this.socket = new WebSocket(this.END_POINT + uuidLink);
-    this.socket.onmessage = (event: MessageEvent) => {
-      this.progressMessages.next(event);
-    };
-    this.socket.onerror = (error: ErrorEvent) => {
-      this.progressMessages.error(error);
-    };
-    this.socket.onclose = () => {
-      this.progressMessages.complete();
-    };
-    return this.socket;
+    const socket = new WebSocket(this.END_POINT + uuidLink);
```

Comment:
Out of curiosity: how is the data received from this websocket being handled since you removed the call to this.progressMessages?

Reply:
@gastaldi this is handled here
https://github.com/fabric8-launcher/ngx-launcher/blob/master/src/app/launcher/create-app/project-progress-createapp-nextstep/project-progress-createapp-nextstep.component.ts#L42

Reply:
I have made it simpler now :-)

Reply:
Awesome, good job